.github/workflows/lint_and_test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install mappymatch black flake8 mypy isort types-requests pytest
+        pip install -r requirements-tests.txt
```

Comment:
If desired, change this to `pip install -e .[tests]` to unpin test environment requirements.